Sources/SwiftKueryORM/Model.swift

```diff
@@ -19,6 +19,18 @@ import KituraContracts
 import Foundation
 import Dispatch
 
+/// Defines the supported formats for persisiting properties of type `Date`.
+public enum DateEncodingFormat {
+    // time - Corresponds to the `time` column type
```

Comment:
If we intend to have these comments (describing each case) to be a part of the API docs, they'd need to start with `///`

Reply:
Also, I think we must document these in the API docs :)